vocab/credentials/v2/vocabulary.yml

```diff
@@ -17,160 +17,146 @@ ontology:
-    comment: A Status List 2021 Entry provides issuers with a mechanism to provide status information for verifiable credentials.
-
-  - id: StatusList2021Credential
-    label: Status List 2021 Credential
-    comment: A Status List 2021 Credential provides issuers with a mechanism to publish a verifiable credential which encapsulates a status list.
-
-  - id: CredentialEvidence
-    label: Credential evidence
-    comment: A Credential Evidence scheme provides enough information to a verifier to determine whether the evidence gathered meets their requirements for issuing a credential. The precise content of each evidence scheme is determined by the specific evidence type definition.
+    deprecated: true
+    comment: A Status List 2021 Entry provides issuers with a mechanism to provide status information for verifiable credentials. <b><em>Placeholder for now; this item may change to something else.</em></b>
 
   - id: VerifiableCredential
+    defined_by: https://www.w3.org/TR/vc-data-model-2.0/#credentials
     label: Verifiable credential
-    comment: |
-      A Credential is a set of one or more claims made by an issuer. A Verifiable Credential is a tamper-evident credential that has authorship that can be cryptographically verified. Verifiable Credentials can be used to build Verifiable Presentations, which can also be cryptographically verified.
 
   - id: VerifiableCredentialGraph
     label: Verifiable credential graph
-    comment: Instances of this class are RDF Graphs, where each of these graphs must include exactly one Verifiable Credential
+    comment: Instances of this class are <a href="https://www.w3.org/TR/rdf12-concepts/#section-rdf-graph">RDF Graphs</a> [[RDF12-CONCEPTS]], where each of these graphs must include exactly one <a href="#VerifiableCredential">Verifiable Credential</a>.
 
   - id: VerifiablePresentation
+    defined_by: https://www.w3.org/TR/vc-data-model-2.0/#presentations
     label: Verifiable presentation
-    comment: |
-      A Presentation is data derived from one or more Credentials, issued by one or more `issuers`, that is shared with a specific `verifier`. A Verifiable Presentation is a tamper-evident Presentation encoded in such a way that authorship of the data can be trusted after a process of cryptographic verification. Certain types of verifiable presentations might contain data that is synthesized from, but do not contain, the original verifiable credentials (for example, zero-knowledge proofs).
+    comment: A Presentation is data derived from one or more Credentials, issued by one or more `issuers`, that is shared with a specific `verifier`. A Verifiable Presentation is a tamper-evident Presentation encoded in such a way that authorship of the data can be trusted after a process of cryptographic verification. Certain types of verifiable presentations might contain data that is synthesized from, but do not contain, the original verifiable credentials (for example, zero-knowledge proofs).
```

Comment:
With better understanding of the overall changes made by this PR, and noting that this term links back to its normative definition, I'm OK with dropping the `comment`.
```suggestion
```